### ISSUE TITLE
App-Version, Release Notes, Change Log

### ISSUE THREAD

**Issue by iantrieschmann** [verdigado/gruene_app#234]
**As an app user, I want to be able to find and read information on the current app version, release notes, and a change log so I can better understand what has been improved in the app and when this was done.**

**Comment by JuliaFreitagBGSt** [comment]
@steffenkleinle can I close this as completed? :)

**Comment by steffenkleinle** [comment]
> [@steffenkleinle](https://github.com/steffenkleinle) can I close this as completed? :)

@JuliaFreitagBGSt I think this depends on whether you think that the users will find the change log/release information on github. We could also just link to the corresponding release notes of that version (or the general releases) in the settings. What do you think?

**Comment by steffenkleinle** [comment]
> I really like the idea [@steffenkleinle](https://github.com/steffenkleinle)!
> 
> Maybe we could make it so that the version info in the settings (e.g. currently the text _Version 2025.7.2 (20530)_ is clickable in the future and leads to the GitHub page?

Yes, that would definitely work. Would you link to the releases overview page (https://github.com/verdigado/gruene-app/releases) or an individual release (e.g. [2025.7.1](https://github.com/verdigado/gruene-app/releases/tag/2025.7.1)? I think the overview page would actually be better as it gives a better overview (due to minor releases that are never on production).

> The only downside is that there are then two links directly below each other (because of the logout button). Maybe we can adapt the design of the logout button so that it looks like the “Aktualisieren”-button in the 2FA and the version button then looks like the “App als zweiten Faktor löschen” button?

I already replaced the button in #689, see https://github.com/verdigado/gruene-app/pull/689#issuecomment-3052428928. Would look like this:

<img width="300" src="https://github.com/user-attachments/assets/8c361858-bf42-4c87-8b86-2e65c1576d65" />

**Comment by JuliaFreitagBGSt** [comment]
> Yes, that would definitely work. Would you link to the releases overview page (https://github.com/verdigado/gruene-app/releases) or an individual release (e.g. [2025.7.1](https://github.com/verdigado/gruene-app/releases/tag/2025.7.1)? I think the overview page would actually be better as it gives a better overview (due to minor releases that are never on production).

Agreed! Link to the overview page :)


> I already replaced the button in [#689](https://github.com/verdigado/gruene-app/pull/689), see [#689 (comment)](https://github.com/verdigado/gruene-app/pull/689#issuecomment-3052428928).

Awesome! But now we need to change it again slightly, so that the primary button (Abmelden) is dark green and the secondary button (version) is light. As it is in the 2FA-section. Right?
